src/components/Map.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import GoogleMapReact from 'google-map-react';
+
+function Map(){
+    const defaultLocation = {
+        center:{
+            lat: 59.95,
+            lng: 30.33
+        },
+        zoom: 11
+
+    };
+    return (
+        <div style={{ height: '100vh', width: '100%' }}>
+          <GoogleMapReact
+            bootstrapURLKeys={{ key:'AIzaSyA_jF-TPUl8qTMZ3BKFTrFOolH9wR7NOz4'}}
+            defaultCenter={defaultLocation.center}
+            defaultZoom={defaultLocation.zoom}
```

Comment:
I love the way these properties have been included as object values. It makes it really easy to read and understand, as well as manage.

Reply:
Agreed, very clean and clear!